astronomer/providers/amazon/aws/hooks/base_aws.py

```diff
@@ -34,6 +34,12 @@ class AwsBaseHookAsync(AwsBaseHook):
 
     async def get_client_async(self) -> AioBaseClient:
         """Create an Async Client object to communicate with AWS services."""
+        # Try to use airflow provider client implementation
+        try:
+            return self._get_aioclient()
```

Comment:
Let's also include assume role config in our aws connection so that we can test it

Reply:
I added some  https://github.com/apache/airflow/pull/28850/files#diff-4e7fd0a0e31b60e534f7cab71ee4d3591c0758c1e76054f2403b6669546cb2d2 do you want me to replicate the same here

Reply:
I was looking for an integration test for `assume_role` functionality

Reply:
created connection in cloud account and added a operator please take a look

Reply:
@pankajastro Could not find it on our deployments. Can you please let me know where this is added

Reply:
It was https://org-airflow-team-org.astronomer.run/dfi0rona/connection/list/